doc/syslog/syslog-design.md

```diff
@@ -1,64 +1,83 @@
-# SONiC Syslog Source IP
-
-## High Level Design document
-
-## Table of contents
-
-- [Revision](#revision)
-- [About this manual](#about-this-manual)
-- [Scope](#scope)
-- [Abbreviations](#abbreviations)
-- [1 Introduction](#1-introduction)
-    - [1.1 Feature overview](#11-feature-overview)
-    - [1.2 Requirements](#12-requirements)
-        - [1.2.1 Functionality](#121-functionality)
-        - [1.2.2 Command interface](#122-command-interface)
-        - [1.2.3 Error handling](#123-error-handling)
-        - [1.2.4 Event logging](#124-event-logging)
-- [2 Design](#2-design)
-    - [2.1 Overview](#21-overview)
-    - [2.2 Syslog Forwarding Output Module](#22-syslog-forwarding-output-module)
-        - [2.2.1 Overview](#221-overview)
-        - [2.2.2 IpFreeBind](#222-ipfreebind)
-    - [2.3 Configuration agent](#23-configuration-agent)
-        - [2.3.1 Overview](#231-overview)
-        - [2.3.2 SSIP parameters](#232-ssip-parameters)
-        - [2.3.3 SSIP configuration](#233-ssip-configuration)
-            - [2.3.3.1 VRF/Source: `unset/unset`](#2331-vrfsource-unsetunset)
-            - [2.3.3.2 VRF/Source: `unset/set`](#2332-vrfsource-unsetset)
-            - [2.3.3.3 VRF/Source: `set/unset`](#2333-vrfsource-setunset)
-            - [2.3.3.4 VRF/Source: `set/set`](#2334-vrfsource-setset)
-    - [2.4 DB schema](#24-db-schema)
-        - [2.4.1 Config DB](#241-config-db)
-            - [2.4.1.1 SSIP table](#2411-ssip-table)
-        - [2.4.2 Data sample](#242-data-sample)
-        - [2.4.3 Configuration sample](#243-configuration-sample)
-        - [2.4.4 Configuration migration](#244-configuration-migration)
-    - [2.5 Flows](#25-flows)
-        - [2.5.1 SSIP add](#251-ssip-add)
-        - [2.5.2 SSIP remove](#252-ssip-remove)
-    - [2.6 CLI](#26-cli)
-        - [2.6.1 Command structure](#261-command-structure)
-        - [2.6.2 Usage examples](#262-usage-examples)
-            - [2.6.2.1 Config command group](#2621-config-command-group)
-            - [2.6.2.2 Show command group](#2622-show-command-group)
-    - [2.7 YANG model](#27-yang-model)
-    - [2.8 Warm/Fast boot](#28-warmfast-boot)
-- [3 Test plan](#3-test-plan)
-    - [3.1 Unit tests via VS](#31-unit-tests-via-vs)
-    - [3.2 Data plane tests via PTF](#32-data-plane-tests-via-ptf)
-
-## Revision
-
-| Rev | Date       | Author         | Description     |
-|:---:|:----------:|:--------------:|:----------------|
-| 0.1 | 18/04/2022 | Nazarii Hnydyn | Initial version |
-
-## About this manual
+# 1. SONiC Syslog Source IP
+
+## 1.1. High Level Design document
+
+## 1.2. Table of contents
+
+- [1. SONiC Syslog Source IP](#1-sonic-syslog-source-ip)
+  - [1.1. High Level Design document](#11-high-level-design-document)
+  - [1.2. Table of contents](#12-table-of-contents)
```

Comment:
We need to get rid of section numbers here and remove them from table of content.
You can use "<!-- omit in toc -->"  directive. See [here](https://github.com/sonic-net/SONiC/pull/1211)